src/main/groovy/edu/oregonstate/mist/locations/mapper/LocationMapper.groovy

```diff
@@ -0,0 +1,28 @@
+package edu.oregonstate.mist.locations.mapper
+
+import edu.oregonstate.mist.locations.core.Location
+import org.skife.jdbi.v2.StatementContext
+import org.skife.jdbi.v2.tweak.ResultSetMapper
+
+import java.sql.ResultSet
+import java.sql.SQLException
+
+class LocationMapper implements ResultSetMapper<Location> {
+    public Location map(int i, ResultSet rs, StatementContext sc) throws SQLException {
+        new Location(
+            id: rs.getInt("id"),
```

Comment:
This is more of a stylistic suggestion but I think using some of the syntatic sugar Groovy offers can aid in asserting mapper correctness visually at the very least.
Here are two examples.
https://github.com/osu-mist/RestPR/blob/e83e06603b68674b03a201a3b676955bef595be7/src/main/groovy/edu/oregonstate/mist/restpr/mapper/UserMapper.groovy

https://github.com/osu-mist/dropwizardtest/blob/feature/co-111-skeleton/src/main/groovy/edu/oregonstate/mist/dropwizardtest/mapper/JobMapper.groovy